### DISCUSSION TITLE
v2.1.0

### DISCUSSION THREAD

**Discussion by joschi** [dropwizard/dropwizard#5317] [Announcements]
## Release Highlights

Dropwizard 2.1.0 comes with the latest and greatest from the Dropwizard team and countless contributors from the community. ❤️

- [dropwizard-health](https://www.dropwizard.io/en/release-2.1.x/manual/core.html#health) is now a core module.
  - See https://github.com/dropwizard/dropwizard-health for a bundle working with Dropwizard 2.0.x)
- Dropwizard Metrics 4.2.x
- Jackson 2.13.x
  - The [Jackson Afterburner](https://github.com/FasterXML/jackson-modules-base/tree/jackson-modules-base-2.13.3/afterburner#readme) module has been replaced by [Jackson Blackbird](https://github.com/FasterXML/jackson-modules-base/tree/jackson-modules-base-2.13.3/blackbird#readme).
- Liquibase 4.x
- Hibernate ORM 5.6.x
- Support for `DualSessionFactory` to split writes and reads into different database sessions.
- Hibernate Validator 6.2x
- Jersey 2.35
- argparse4j 0.9.x

## Upgrade Notes

Please refer to the upgrade notes in the Dropwizard documentation at https://www.dropwizard.io/en/release-2.1.x/manual/upgrade-notes/upgrade-notes-2_1_x.html.

## Release MVPs

Special thanks to  Russell Howe (@rhowe), Steffen Nießing (@zUniQueX),  Natalie Zamani (@natalie-zamani), and Peter Stackle (@pstackle) for their invaluable contributions to the release of Dropwizard 2.1.0! ❤️

Please refer to https://github.com/dropwizard/dropwizard/releases/tag/v2.1.0 for the complete release notes of Dropwizard 2.1.0!

**Comment by rhowe** [comment]
No credits for @dependabot and @renovate-bot?!

**Reply by joschi** [reply]
Only credits for people made of flesh and blood. 😉
